<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UiSettingsPage</class>
 <widget class="QWidget" name="UiSettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="toggleCorridorTime">
     <property name="text">
      <string>Toggle corridor time</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Sound</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="2">
       <widget class="QPushButton" name="btPlayCardInserted">
        <property name="icon">
         <iconset resource="../startertool.qrc">
          <normaloff>:/images/play.svg</normaloff>:/images/play.svg</iconset>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="btPlayCorridorTimeError">
        <property name="icon">
         <iconset resource="../startertool.qrc">
          <normaloff>:/images/play.svg</normaloff>:/images/play.svg</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="btPlayCardNotFound">
        <property name="icon">
         <iconset resource="../startertool.qrc">
          <normaloff>:/images/play.svg</normaloff>:/images/play.svg</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="lstSoundCardInserted"/>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="lstSoundCardNotFound"/>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="lstSoundCorridorTimeError"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Card inserted</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Card not found</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Corridor time error</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../startertool.qrc"/>
 </resources>
 <connections/>
</ui>
